src/interpreter.rs

```diff
@@ -642,18 +722,17 @@ pub fn eval_step<'a>(
                             // Thus as a special case we tell `ScriptNum` to accept up
```

Comment:
```suggestion
                            // Thus as a special case we tell `parse_num` to accept up
```

Reply:
Resolving as a nit so that this can merge.